### PR TITLE
Update download.mdx

### PR DESCRIPTION
**Pull request by uSlackr**
Added Restart Ha step

### CODE REVIEW

File: documentation/setup/download.mdx [hunk 1]
```diff
@@ -21,6 +21,7 @@ If you are not sure what type of Home Assistant you are using or how to find tha
 1. Start the SSH add-on
 1. Connect to the SSH add-on
 1. Run the HACS download script
+1. After the script completes, restart Home Assistant
```
Comment (frenck) [inline documentation/setup/download.mdx:24]: This does not only apply to OS/Supervised, right? Container installs need to restart their container and Core install need to restart as well.

**Review by frenck — DISMISSED** [review]
Makes sense, @uSlackr 👍 

Let's make sure we add it to the other installation methods as well.

../Frenck

**Comment by ludeeus** [comment]
@coderabbitai review

File: documentation/setup/download.mdx [hunk 2]
```diff
@@ -21,18 +21,19 @@ If you are not sure what type of Home Assistant you are using or how to find tha
 1. Start the SSH add-on
 1. Connect to the SSH add-on
 1. Run the HACS download script
-
+1. After the script completes, restart Home Assistant
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply documentation/setup/download.mdx:24]: That has already been handled 👍

File: documentation/setup/download.mdx [hunk 3]
```diff
@@ -44,11 +45,12 @@ wget -O - https://get.hacs.xyz | bash -
 1. Open a terminal
 1. Change to the user that is running Home Assistant
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply documentation/setup/download.mdx:46]: That is a good suggestion, but not for this PR.

**Review by ludeeus — APPROVED** [review]
Thanks 👍

**Pull request merged**